terraform/ci/variables.tf

```diff
@@ -44,3 +44,30 @@ variable "ci_iam_roles" {
   ]
   description = "List of IAM roles needed to run integration tests included in CI/CD."
 }
+
+variable "region" {
+  description = "The default Google Cloud region to deploy resources in (defaults to 'us-central1')."
+  type        = string
+  default     = "us-central1"
+}
+
+variable "jvs_container_image" {
+  description = "Container image for the jvsctl CLI and server entrypoints."
+  type        = string
+}
+
+variable "plugin_envvars" {
```

Comment:
both "plugin_envvars" and "ci_service_account_email" variable should be removed. We wont test plugin in jvs ci.

Reply:
Moved.

Reply:
oops, missed two more: variable "ci_iam_roles", "registry_repository_id"